tools/project_manager/page_table.inc

```diff
@@ -383,7 +385,6 @@ function echo_page_table(
             echo "<td>$clear_string</td>";
 
             // Bad Page
```

Comment:
Let's move the comment along with the line and make sure there's a space between this comment and the comment block on line 350.